src/index.js

```diff
@@ -27,6 +27,9 @@ async function publishConfig(params) {
     params.__ow_logger,
     params.indexconfig,
     params.algoliaappid,
+    params.wskAuth,
+    params.wskHost,
+    params.wskNamespace,
```

Comment:
please add fallback to:

- `process.env.__OW_API_KEY`
- `process.env.__OW_NAMESPACE`
- `process.env.__OW_API_HOST`

then, we don't need to pass the creds in `helix-cli`. see https://github.com/adobe/helix-cli/issues/1378#issuecomment-616878803